mmv1/third_party/terraform/tests/data_source_google_firebase_web_app_test.go.erb

```diff
@@ -0,0 +1,58 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceGoogleFirebaseWebApp(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"project_id":   getTestProjectFromEnv(),
+		"display_name": "Display Name WebApp DataSource",
```

Comment:
This actually needs to be prepended with `tf-test` now, so that it is cleaned up by our sweepers.

Reply:
Done.